spec/missing_keys_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'MissingKeys' do
+  describe '#required_plural_keys_for_locale(locale)' do
+    let(:task) { ::I18n::Tasks::BaseTask.new }
+
+    def configuration_from(locale)
+      {
+        "#{locale}": {
+          i18n: {
+            plural:{
+              keys: [
+                :one,
+                :other
+              ],
+            rule: lambda {}
```

Comment:
Is this `rule` necessary for the test?

Reply:
It's not necessary for the test, but it is included in the eval from `load_rails_i18n_pluralization!(locale)`. I can remove it if you'd prefer.